backend/open_webui/routers/auths.py

```
@@ -514,42 +713,101 @@ async def signup(request: Request, response: Response, form_data: SignupForm):
-                                detail="Failed to fetch OpenID configuration",
-                            )
-            except Exception as e:
-                raise HTTPException(status_code=500, detail=str(e))
-
-    return {"status": True}
+        log.error(f"Signup error: {str(err)}")
+        raise HTTPException(500, detail="An internal error occurred during signup.")
+
+
+@router.get("/prepare-logout")
+async def prepare_logout(request: Request, response: Response, user=Depends(get_current_user)):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)